src/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Space Wiki</title>
+    <!-- CSS only -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+    <script src="https://kit.fontawesome.com/69bb18a36d.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header class="navbar fixed-top navbar-light bg-light">
+        <i class="fas fa-user-astronaut"></i>
+        <a href="#">Astronauts in Space <span id="count-items"></span></a>
+        <a href="#">My Planets</a>
+        <a href="#">Races</a>
+    </header>
```

Comment:
- [ ] Currently, you did not implement the `like button` functionality. When a user clicks on the Like button of an item, the interaction should get recorded in the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270), and the screen gets updated. I would suggest that you implement it  as it is an important part of `user interaction` of your app

![image](https://user-images.githubusercontent.com/98453979/205461160-17e7bc83-b5b7-4635-9b43-1328f88830d8.png)

Reply:
Hi, 
You need to set up and .env for the Api key .
![image](https://user-images.githubusercontent.com/64089437/205461827-23a2c6fc-a630-4403-9b1e-c25f8c534eb6.png)